assets/js/calypsoify.js

```diff
@@ -11,10 +11,15 @@
         }
     } );
 
+// TODO Remove the notice handlers after https://github.com/Automattic/jetpack/pull/13126 has been released.
```

Comment:
Minor nitpick: I think we should update all the todo blocks to the new standardized format of `@todo Description of todo.`

Also I believe this line will need to be indented one tab.

Reply:
> Duplicate notices (plugin page, possibly hidden notices being shown?)

That happens if your browser is using the old version of `calypsoify.js` in this repo. The updated script checks to see whether the Calypso markup has already been added.

> Minor nitpick: ... `@todo`

I'll fix those up.

Reply:
@joshuatf This is ready for another review.